apps/condo/domains/contact/access/ExportContactsService.js

```diff
@@ -0,0 +1,27 @@
+const { throwAuthenticationError } = require('@condo/domains/common/utils/apolloErrorFormatter')
+const { checkOrganizationPermission, checkRelatedOrganizationPermission } = require('@condo/domains/organization/utils/accessSchema')
+const { Organization } = require('@condo/domains/organization/utils/serverSchema')
+const get = require('lodash/get')
+
+async function canExportContactsToExcel ({ args: { data: { where } }, authentication: { item: user }, context }) {
```

Comment:
This way it would be a bit readable

```
async function canExportContactsToExcel (params) {
    const { 
        args: { data: { where } }, 
        authentication: { item: user }, 
        context 
    } = params
    const organizationId = get(where, 'organization.id')
    
    if (!user) return throwAuthenticationError()
    if (user.isAdmin) return true
    if (!organizationId) {
        const organizationFromWhere = get(where, 'organization')
        const [relatedFromOrganization] = await Organization.getAll(context, organizationFromWhere)

        if (!relatedFromOrganization) return false

        const canManageRelatedOrganizationContacts = await checkRelatedOrganizationPermission(context, user.id, relatedFromOrganization.id, 'canManageContacts')

        if (canManageRelatedOrganizationContacts) return true

        return false
    }

    return await checkOrganizationPermission(context, user.id, organizationId, 'canManageContacts')
}
```

Reply:
codestyled in previous commit :)
dbcf4e8

Reply:
If we move arguments to `params` we will lose code suggestions in IDE.
Anyway, I fixed some codestyle issues and improved the code a bit.

Reply:
@akarjakin please use suggestion button

Reply:
Usually we don’t extract the paeans inside the function. It’s look like a new style